js-intro.js

```diff
@@ -56,6 +102,26 @@ var notAString2 = 42
 var toonimals = [ {name: "Itchy", animal: "mouse"}, {name: "Stimpy", animal: "cat"}, {name: "Daffy", animal: "duck"}, {name: "Scratchy", animal: "cat"}, {name: "Ren", animal: "dog"}, {name: "Felix", animal: "cat"}]
 
 
+onlyCats = (array) => {
+  let newArr = array.filter(value=> value.animal==="cat")
+  return newArr
```

Comment:
Same as above, you can remove a line of code by just returning the filter outright :)